lib/schedule.js

```diff
@@ -348,9 +348,9 @@ function recurMatch(val, matcher) {
 
   if (typeof matcher === 'number' || typeof matcher === 'string') {
     return (val === matcher);
-  } else if (typeof matcher === 'object' && matcher instanceof Range) {
+  } else if (matcher instanceof Range) {
     return matcher.contains(val);
-  } else if (Array.isArray(matcher) || (typeof matcher === 'object' && matcher instanceof Array)) {
+  } else if (Array.isArray(matcher) || (matcher instanceof Array)) {
```

Comment:
Out of curiosity: `Array.isArray` can be `false` while `instanceof Array` `true`?